runtime/api/Robot.py

```diff
@@ -12,49 +12,494 @@
 >>> from api import Robot
 '''
 import memcache
-
 memcache_port = 12357
 mc = memcache.Client(['127.0.0.1:%d' % memcache_port]) # connect to memcache
 
 motor = {}
 
+def _lookup(name): #Returns actual UID given name
+    return name
+
 def get_motor(name):
     """Returns the current power value for a motor.
 
     :param name: A string that identifies the motor
     :returns: A value between -100 and 100, which is the power level of that motor.
+
+    :Examples:
+
+    >>> motor = get_motor(motor1)
+
     """
     name_to_value = mc.get('motor_values')
-    if name in name_to_value:
+    assert type(name) is str, "Type Mismatch: Must pass in a string"
+    try:
         return name_to_value[name]
-    return 'Error, motor with that name not found'
+    except KeyError:
+        raise KeyError("Motor name not found.")
 
 def set_motor(name, value):
+
     """Sets a motor to the specified power value.
 
-    :param name: A string that identifies the motor
-    :param value: A decimal value between -100 and 100, the power level you want to set
+    :param name: A string that identifies the motor.
+    :param value: A decimal value between -100 and 100, the power level you want to set.
```

Comment:
Likewise, I suggest using -1.0 to 1.0. This will probably involve implementing a conversion factor somewhere